pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -45,10 +44,12 @@ func Register(plugins *admission.Plugins) {
 				glog.Infof("Admission plugin %q is not configured so it will be disabled.", api.PluginName)
 				return nil, nil
 			}
-			return newClusterResourceOverride(pluginConfig)
+			return newClusterResourceOverride(pluginConfig, defaultGetNamespaceLimitRanges)
```

Comment:
this is an odd injection pattern... why not have a LimitRangeLister in the plugin and populate it like this:

```
// SetInternalKubeInformerFactory implements the WantsInternalKubeInformerFactory interface.
func (p *clusterResourceOverridePlugin) SetInternalKubeInformerFactory(f informers.SharedInformerFactory) {
	p.limitRangesLister = f.Core().InternalVersion().LimitRanges().Lister()
}
```